src/test/java/com/nvidia/spark/rapids/jni/DecimalUtilsTest.java

```diff
@@ -192,9 +193,9 @@ void simplePosDivOneByZero() {
     try (ColumnVector lhs =
              makeDec128Column("1.0", "10.0", "1.0", "1000000000000000000000000000000000000.0");
          ColumnVector rhs =
-             makeDec128Column("1",   "2",    "0",   "5");
+             makeDec128Column("1", "2", "0", "5");
          ColumnVector expectedBasic =
-             makeDec128Column("1.0", "5.0",  "0",   "200000000000000000000000000000000000.0");
+             makeDec128Column("1.0", "5.0", "0", "200000000000000000000000000000000000.0");
```

Comment:
Nit: The whitespace being removed here was for readability to line up the row data across columns, and this unrelated change makes it arguably harder to read.